sql/moz-fx-data-shared-prod/adjust/adjust_derived/view.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE VIEW
```

Comment:
The CI appears to fail due to the folder structure where this view is found.

The folder should be:
`sql/moz-fx-data-shared-prod/adjust/view.sql`

Instead of:
`sql/moz-fx-data-shared-prod/adjust/adjust_derived/view.sql`